tests/ga/test_extension.py

```diff
@@ -1170,11 +1170,166 @@ def test_get_ext_handling_status(self, *args):
 
         os.path.exists = orig_state
 
+    @patch("azurelinuxagent.ga.exthandlers.ExtHandlerInstance.get_status_file_path")
+    def test_collect_ext_status(self, patch_get_status_file_path, *args):
+        """
+        Testing is_ext_handling_complete() with various input and
```

Comment:
can we have a better description of this test. Current description says it tests 'is_ext_handling_complete' with some inputs checking some outputs... no idea what that does :)